README.md

```
@@ -171,6 +171,46 @@ Fixes lint errors in TypeScript files.
+   -c, --content <text>          email text content or message enclosed in double-quotes
+   -r, --recipients [emails...]  list of email addresses separated by whitespace or comma
+   -e, --env <path>              path to .env file (optional)
+   -h, --help                    display help for command
+   ```
+
+### `npm run send-email:dev`
+
+- Sends an email using the command line interface (CLI) in development mode using TypeScript.
+- Append a double dash `--` to pass arguments to the CLI commands.
+- Usage: view the `"npm run send-email"` script for more information. They share similar usage.
+   - > 💡 **NOTE:** Append `:dev` in the script eg., `npm run send-email:dev`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)